src/components/ContentNode/Caption.vue

```diff
@@ -0,0 +1,87 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2021-2023 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+<template>
+  <component class="caption" :class="{ centered }" :is="tag">
+    <template v-if="title">
+      <strong>{{ title }}</strong>&nbsp;<slot />
+    </template>
+    <template v-else>
+      <slot />
+    </template>
+  </component>
+</template>
+
+<script>
+const CaptionTag = {
+  caption: 'caption',
+  figcaption: 'figcaption',
+};
+
+export default {
+  name: 'Caption',
+  constants: { CaptionTag },
+  props: {
+    title: {
+      type: String,
+      required: false,
+    },
+    centered: {
```

Comment:
Maybe. It is more clear but also kind of an odd property name. I'll think on something better.

Reply:
https://github.com/apple/swift-docc-render/pull/706/commits/54a6137a9bb80c3789cac7bb35cf52caa40c498f

I think the position makes more sense as a prop and it's just an implementation detail that we center the bottom positioned one. I considered leaving out the prop entirely since it is always derived from the presence of the title, but I wanted it to be slightly more flexible in case we wanted to use it anywhere else.

Reply:
Looks good!